src/components/Copyable.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+    <div class="copyable" :class="{ copied }" @click="copy">
+        <div class="background"></div>
+        <slot></slot>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from 'vue-property-decorator';
+import { Clipboard } from '@nimiq/utils';
+
+/**
+ * **Copyable**
+ *
+ * Copyable can be used to make a click on one or more elements copy content to the Clipboard with visual feedback.
+ * By default the children's contents are copied to the Clipboard. Alternatively, a specific text to be copied can be
+ * provided.
+ *
+ * Props:
+ *
+ * **text** {string} [optional] - A specific text to be copied to the clipboard
+ */
+@Component
+export default class Copyable extends Vue {
+    @Prop(String)
+    public text?: string;
+
+    public copy() {
+        // Note that when iterating over childNodes, pseudo elements are excluded.
+        const text = this.text || [ ...this.$el.childNodes ].reduce((concatenated, node) => {
+            let nodeText = '';
+            if (node.nodeType === Node.TEXT_NODE) {
+                nodeText = node.textContent.trim();
+            } else if (node.nodeType === Node.ELEMENT_NODE) {
+                // use innerText as it better represents the visible content than textContent
+                nodeText = (node as HTMLElement).innerText;
+            }
+
+            if (concatenated && nodeText) {
+                concatenated += '\n';
+            }
+            concatenated += nodeText;
+            return concatenated;
+        }, '');
+        Clipboard.copy(text);
+
+        window.clearTimeout(this._copiedResetTimeout);
+        this.copied = true;
+        this._copiedResetTimeout = window.setTimeout(() => {
+            this.copied = false;
+        }, 1800);
+    }
+
+    private copied: boolean = false;
+    private _copiedResetTimeout: number | null = null;
```

Comment:
I believe private variables should be defined at the top of a class, before public methods.

(Also, one private variable has a leading underscore and the other doesn't? :wink:)

Reply:
It's not included in the linting rules for this repo, but indeed how the hub is configured.
Once we have our stuff merged, we should consider to finally establish unified linting rules across repositories.